marshal.go

```diff
@@ -173,14 +174,19 @@ func (x *GoSNMP) sendOneRequest(packetOut *SnmpPacket,
 			return nil, err
 		}
 
-		// Request ID is an atomic counter (started at a random value)
-		reqID := atomic.AddUint32(&(x.requestID), 1) // TODO: fix overflows
+		reqID := atomic.AddUint32(&(x.requestID), 1)
```

Comment:
Should we preserve the comment here? It does seem a little out of place, as there's nothing about randomization here.

Reply:
should I just add ``// Request ID is an atomic counter`` instead of removing the entire comment?

Reply:
How about this: S
```suggestion
		// Request ID is an atomic counter that wraps to 0 at max int32.
		reqID := atomic.AddUint32(&(x.requestID), 1)
```

Reply:
That looks better indeed :), will update the PR accordingly

Reply:
Done